hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/DirectoryDeletingService.java

```diff
@@ -238,6 +295,7 @@ public BackgroundTaskResult call() {
         }
       }
       // place holder by returning empty results of this call back.
+      taskCount.getAndDecrement();
```

Comment:
Better to do `taskCount.getAndDecrement();` in finally. I some en case of some exception it doesn't get decremented then from next time complete deletingservice will halt.

Reply:
Done